### PR TITLE
add ansible-collection galaxy.yml and according test

### PR DESCRIPTION
**Pull request by rndmh3ro**

### CODE REVIEW

File: galaxy.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+### REQUIRED
+
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric characters and underscores. Additionally namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: ara
```
Comment (dmsimard) [inline galaxy.yml]: This might need to be ansible-community ? I'll find out if it's okay to publish from here.

File: galaxy.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: ara
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 1.0.0
+
+# The path to the Markdown (.md) readme file. This path is relative to the root of the collection
+readme: README.md
```
Comment (dmsimard) [inline galaxy.yml]: Needs to be README.rst

File: galaxy.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric characters and underscores. Additionally namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: ara
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: ara
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 1.0.0
```
Comment (dmsimard) [inline galaxy.yml]: For the sake of simplicity I would sync this version number with the one from ara -- the current latest release is 1.1.0

**Comment by dmsimard** [comment]
Just a few comments but I'm happy with how simple getting this working turned out to be.

I'd love to merge this but we aren't set up to accept pull requests yet: https://ara.readthedocs.io/en/latest/contributing.html#issues-and-pull-requests

The documentation explains how to submit a patch to gerrit instead.
Let me know if you'd like to do it or I can also send the patch on your behalf.

Happy to help if you hop on [IRC or Slack](https://ara.recordsansible.org/community/).

**Comment by rndmh3ro** [comment]
I'll address the issues and submit a PR to gerrit!

**Comment by rndmh3ro** [comment]
https://review.opendev.org/#/c/688312/

**Pull request closed without merging**